app/content/drill/[id]/leaderboard.js

```diff
@@ -177,10 +178,18 @@ export default function Leaderboard() {
             >
               <List.Item
                 title={userInfo[userId] ? userInfo[userId]["name"] : "Unknown"}
-                left={() => <Avatar.Text size={24} label="XD" />}
+                left={() => (
+                  <Avatar.Image
+                    size={24}
+                    source={{ uri: userInfo[userId]["pfp"] }}
+                  />
+                )}
                 right={() => (
                   <View style={{ flexDirection: "row", alignItems: "center" }}>
-                    <Text>{numTrunc(attempt["value"], true)} ft</Text>
+                    <Text>
+                      {numTrunc(attempt["value"], true)}{" "}
+                      {drillInfo.aggOutputs[mainOutputAttempt].distanceMeasure}
```

Comment:
For the leaderboard that have strokes gained. Do we want to have SG? not a distenceMeasure but is a unit
![image](https://github.com/Golf-Drill-Challenge-App/Golf-App/assets/91503842/d55ed5cd-3273-4e6b-bf40-b3346ca1f439)

Reply:
The issue here could be that this looks too empty. Maybe make the elements bigger?

Reply:
It will have a bit more once the numbers are added, would a divider look good between each of the entries?